app/controllers/posts_controller.rb

```diff
@@ -50,6 +50,21 @@ def interactions
     end
   end
 
+  def mentionable_in_comment
+    respond_to do |format|
+      format.json {
+        if params[:id].present? && params[:q].present?
```

Comment:
> It's not possible the way the mention class is written right now.

I don't see how this shouldn't be possible with the current mention system, you just remove the need to type 2 additional characters.

> Plus, on a personnal point of view, I find this counter intuitive to be able to mention people without a @ on comments and not in publisher.

I don't understand what you mean here? There will be an @. So you type the @ and you already see the last few commenters and the post author as suggestion, but without the need to type the actual name (so you can select them with arrow-keys). I don't see where this should be "counter intuitive"?

> Same problem: no data is available on the front-end right now.

That's why I suggested to use the data from the backend (because it is available there and can be returned easily), and refactor the frontend later to use the data the frontend already know. If the refactoring is easy, then I'd prefer to directly use the data from the frontend, but I can't estimate that.